Automattic/jetpack
# Update/recommendations copy updates

#### Changes proposed in this Pull Request:

* This PR updates some copy and links on the boost and backup recommendations.

#### Other information:


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check out this branch locally and run `jetpack build plugins/jetpack` or checkout this branch using the beta plugin
* Navigate to `/wp-admin/admin.php?page=jetpack#/recommendations/boost`
* Confirm that the copy has been broken into two paragraphs.
* Confirm that the text "Jetpack Boost's" is linked to https://jetpack.com/boost/
![Screen Shot 2022-08-31 at 3 16 08 PM](https://user-images.githubusercontent.com/18016357/187764101-e375de21-796e-41b5-98e2-e4f828a8fa58.png)

* Now, navigate to `/wp-admin/admin.php?page=jetpack#/recommendations/backup-plan`
* Confirm that the text on the CTA button has been updated to read "Learn More About Site Backups"
![Screen Shot 2022-08-31 at 2 02 31 PM](https://user-images.githubusercontent.com/18016357/187753345-59cce44f-6d3f-4239-9300-79c31b4d8793.png)
